Confus/Confus/Player.cpp

```diff
@@ -220,4 +230,19 @@ namespace Confus
     {
         m_SoundEmitter = new Audio::PlayerAudioEmitter(PlayerNode);
     }
+
+     void Player::setConnection(Networking::ClientConnection* a_Connection)
+	{
+        m_Connection = a_Connection;
+	}
+
+     void Player::sendAttackMessageToServer(bool a_IsHeavyAttack, bool a_IsMainPlayer) const
+	{
+        std::unique_ptr<RakNet::BitStream> bitStreamOut = std::make_unique<RakNet::BitStream>();
+        bitStreamOut->Write(static_cast<RakNet::MessageID>(Networking::ClientConnection::EPacketType::PlayerAttack));
+        a_IsHeavyAttack ? bitStreamOut->Write(true) : bitStreamOut->Write(false);
+        a_IsMainPlayer ? bitStreamOut->Write(true) : bitStreamOut->Write(false);
+        
```

Comment:
Why are these conditional statements? You can write the boolean directly (i.e. 'bitStreamOut->Write(a_IsHeavyAttack))